index.html

```diff
@@ -9,187 +9,230 @@
-                </ul>
-                <div class="framework">
-                    <p id="framework-section">Frameworks </p>
-                </div>
-                
-                <div class="framework">
-                <p id="framework-section">Skills</p>
-                </div>
-            </div>
-        </section>
+ <section class="about-self" id="about">
+    <div class="about-self-content">
+        <h2>About</h2>
+        <h2>Myself</h2>
+        <p id="intro-details-foot">I’m a software developer!
+            I can help you build a product, 
+            feature or website Look 
+            through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+            <h3 id="intro-connect">LET'S CONNECT</h3>
+            <ul class="socials">
+                <li><a href=""><img src="img/twitter-icon.png" alt=""></a></li>
+                <li><a href=""><img src="img/linkedin-icon.png" alt="linkedin"></a></li>
+                <li><a href=""><img src="img/medium-icon.png" alt=""></a></li>
+                <li><a href=""><img src="img/github-icon.png" alt=""></a></li>
+                <li><a href=""><img src="img/angellist-icon.png" alt=""></a></li>
+            </ul>
+           
+            <div id="resume"><a class="resume-button" href="#">Get my resume</a></div>
+    </div>
+    <div class="about-self-content">
+        <p id="language-section">Languages</p>
+        <ul class="languages">
+            <li class="language">
+                <img src="img/Ellipse 1.png" alt="JavaScript">
+                <p>JavaScript </p>
+                <a href="#"><i class="lang-arrow right"></i></a>
+            </li>
+            <li class="language">
+                <img src="img/Ellipse 2.png" alt="HTML">
+                <p>HTML</p> 
+                <a href="#"><i class="lang-arrow right"></i><a></a>
+            </li>
+            <li class="language">
+                <img src="img/Ellipse 3.png" alt="CSS">
+                <p>CSS</p> 
+                <a href="#"><i class="lang-arrow right"></i><a>
+            </li>
+        </ul>
+        <div class="framework">
+            <p id="framework-section">Frameworks </p>
+        </div>
+        
+        <div class="framework">
+        <p id="framework-section">Skills</p>
+        </div>
+    </div>                  
+</section>
 
-       <form action="https://formspree.io/f/mzbwrodq" method="post">
+       <form id="contact" action="https://formspree.io/f/mzbwrodq" method="post">
         <div class="form-section">
         <h2 class="form-header">Contact Me</h2>
-        <p>If you have an application you are interested in developing, 
+        <p id="form-details">If you have an application you are interested in developing, 
             a feature that you need built or a project that needs coding. 
             I’d love to help with it </p>
-        <ul>
+        <ul class="inputs">
```

Comment:
Please activate the `hover` effect of the button `Get in touch`. Use the design template guide of figma. This is so that you meet the requirements for this milestone and also so that your page looks better.
<img width="375" alt="Screenshot 2022-09-30 at 13 07 29" src="https://user-images.githubusercontent.com/48546992/193247444-8c56af61-b181-4de9-bd2b-4aaa8e78a5b3.png">